src/course-outline/status-bar/messages.js

```diff
@@ -41,6 +41,14 @@ const messages = defineMessages({
     id: 'course-authoring.course-outline.status-bar.highlight-emails.link',
     defaultMessage: 'Learn more',
   },
+  courseTagsTitle: {
+    id: 'course-authoring.course-outline.status-bar.course-tags',
+    defaultMessage: 'Course tags',
```

Comment:
This is a friendly advance warning that soon we'll be requiring descriptions to be added to messages: https://github.com/openedx/frontend-build/issues/517.  For now I'm not going to block the PR, but it would be nice if you added some. :)

Reply:
Sure! Done: f0001a4a080cffcdbb7c73cc21f999fda9c4423f